### PR TITLE
[MTL-2019] Add bios and abstracts

### PR DESCRIPTION
**Pull request by idcrosby**
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2019] Add Bluth Company as a sponsor`*

### CODE REVIEW

File: content/events/2019-montreal/program/charles-abondo.md [hunk 1]
```diff
@@ -7,4 +7,17 @@ Type = "talk"
...
+In this paper, we will discuss how the rise and prominence of scaled agile is helping CN collaborate and deliver to the business and our customers more safely and efficiently. We will explore the concept of having a strategy around DevSecOps that provides an essential ingredient to growth and productivity. Some of the key questions we propose to answer in this paper cover top of mind concerns like:
+
+How does DevSecOps work in an environment where safety is critical?
+How do analog, digital & modernization coexist?
+How do we bring security to move from on-premise to cloud?
+How do we mobilize our organization and talent to deliver?
+How do we find the right service integrator to partner with?
+
+From design, through the development process to production support, this paper will walk you through CN’s DevSecOps modernization journey and provide fundamental learnings along the way.
+To enable screen reader support, press ⌘+Option+Z To learn about keyboard shortcuts, press ⌘slash
```
Comment (bridgetkromhout) [inline content/events/2019-montreal/program/charles-abondo.md]: Are we sure that `To enable screen reader support, press ⌘+Option+Z To learn about keyboard shortcuts, press ⌘slash` is supposed to be part of this talk description??

File: content/events/2019-montreal/speakers/chuck-mountz-jr.md [hunk 2]
```diff
@@ -7,4 +7,4 @@ linktitle = "chuck-mountz-jr"
 
 +++
 
-
+Chuck is a Sr Manager of Product Engineering at Comcast. He leads a platform services team focused on delivering services and tools to help accelerate delivery for the greater Network Services Engineering organization. Chuck is passionate about people. Specifically organizational culture, developing talent, fostering partnerships, and building enterprise-wide communities. He co-founded the internal DevOpsDays events at Comcast. Chuck is proud that these events have brought together over 900 employees in one of the largest tech communities at Comcast. He is delighted that over 40 employees have had the opportunity to refine their communication skills by presenting at these events.
```
Comment (bridgetkromhout) [inline content/events/2019-montreal/speakers/chuck-mountz-jr.md]: Please talk with Chuck about correcting the wording in this bio, as I already spoke with him and they changed the name of their internal event to "Comcast DevOps Event".

File: content/events/2019-montreal/program/charles-abondo.md (hunk 1 above)
Comment (idcrosby) [reply content/events/2019-montreal/program/charles-abondo.md]: Indeed it should not 😬 . Thanks for catching this

File: content/events/2019-montreal/speakers/chuck-mountz-jr.md (hunk 2 above)
Comment (idcrosby) [reply content/events/2019-montreal/speakers/chuck-mountz-jr.md]: Thanks @bridgetkromhout we've received the updated bio from Chuck

**Pull request merged**